test/models/arrayValidators.ts

```diff
@@ -17,36 +17,36 @@ export enum ArrayValidatorEnumNumber {
 export class ArrayValidators {
   // String-Array Values
   @arrayProp({ items: String, maxlength: 3 })
-  public maxLength?: string[];
+  public maxLength!: string[];
```

Comment:
why are all the properties here changed to `!`?

Reply:
These are test models, I just made them required to not have extra checks for undefined in tests.